src/app/pages/StoryPage/index.jsx

```diff
@@ -148,16 +152,25 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     'data-e2e': 'most-read',
   }))``;
 
+  const SectionLabelWrapper = styled(SectionLabel)`
+    ${({ removeTopMargin }) =>
+      removeTopMargin &&
+      css`
+        margin-top: 0;
+      `}
+  `;
```

Comment:
I don't like the way we have to break encapsulation here to remove the top margin, I think it'd be cleaner if `SectionLabel` supported something like a `topItem` prop; `SectionLabel` taking responsibility for how it looks when it is the top item in the secondary column.

This means that if the css is changed for `SectionLabel` we don't also have update the logic here in the Story page.

Reply:
removed this change as it's already been identified and reported by UX in a standalone issue.